packages/frontend/src/components/UserSettings/MyWarehouseConnectionsPanel/types.ts

```diff
@@ -0,0 +1,6 @@
+// TODO - to be moved to common
+export type Credentials = {
+    name: string;
+    username: string;
+    password: string;
+};
```

Comment:
to be moved to `common` and this will vary depending on the warehouse connection type